zeebe/gateway/src/main/java/io/camunda/zeebe/gateway/RequestMapper.java

```diff
@@ -126,12 +126,22 @@ public static BrokerPublishMessageRequest toPublishMessageRequest(
 
   public static BrokerUpdateJobRetriesRequest toUpdateJobRetriesRequest(
       final UpdateJobRetriesRequest grpcRequest) {
-    return new BrokerUpdateJobRetriesRequest(grpcRequest.getJobKey(), grpcRequest.getRetries());
+    final var brokerRequest =
+        new BrokerUpdateJobRetriesRequest(grpcRequest.getJobKey(), grpcRequest.getRetries());
+    if (grpcRequest.hasOperationReference()) {
+      brokerRequest.setOperationReference(grpcRequest.getOperationReference());
+    }
```

Comment:
❓ Did we consider the case where users can set negative values? Currently all int64 long values are allowed. We could ensure positive values only (similar to other request properties like valid json, see code near the bottom of this class). This might make it easier to identify records with `-1` as those where no operation reference was set at all. But, I'm not sure whether that's useful. There could be value in users setting `-1` themselves. What do you think?

Reply:
It's a good instinct to be cautious, but I honestly couldn't think of a reason why would users do that for an optional field. I think it's unnecessary to check it here.

The other fields that have validations down there are strings, and they have special needs that I don't feel this one have.

Reply:
:wrench: If we keep it encoded as uint64 for SBE, it would be good to do. Otherwise, if we use int64, then we can leave it as is.